guiEditor/src/diagram/properties/containerPropertyComponent.tsx

```diff
@@ -0,0 +1,47 @@
+
+import { Container } from "babylonjs-gui";
+import * as React from "react";
+import { ButtonLineComponent } from "../../sharedUiComponents/lines/buttonLineComponent";
+import { LineContainerComponent } from "../../sharedUiComponents/lines/lineContainerComponent";
+import { TextLineComponent } from "../../sharedUiComponents/lines/textLineComponent";
+import { IContainerComponentProps } from "./propertyComponentProps";
+
+export class ContainerPropertyTabComponent extends React.Component<IContainerComponentProps> {
+    constructor(props: IContainerComponentProps) {
+        super(props);
+    }
+
+    AddElement()

```

Comment:
Missing a public/protected/private modifier and probably the first letter should be lowercase (except if it is a static function?).